subsys/bluetooth/host/gatt.c

```diff
@@ -1275,7 +1343,7 @@ static int gatt_indicate(struct bt_conn *conn,
 			return -EINVAL;
 		}
 
-		value_handle += 1U;
+		handle += 1U;
```

Comment:
I realise this is already in the old code, by why is this not simply `handle++`?

Reply:
Right, will fix this since there is no reason to keep the original code.